### PR TITLE
Support for probe response and client disconnect thresholds

### PR DESCRIPTION
**Pull request by yashvardhan-netex**
Added support to configure probe response and client disconnect thresholds
via ovsdb.

Signed-off-by: Yashvardhan <yashvardhan@netexperience.com>

### CODE REVIEW

File: feeds/wlan-ap/opensync/src/platform/openwrt/src/rrm/src/rrm_channel.c [hunk 1]
```diff
@@ -19,25 +19,32 @@ static struct ev_timer  rrm_nf_timer;
...
+		[WIF_ATTR_CLIENT_DISCONNECT_THRESHOLD] = { .name = "signal_stay", .type = BLOBMSG_TYPE_INT32 },
 };
 
 const struct uci_blob_param_list wifi_config_param = {
 		.n_params = __WIF_ATTR_MAX,
 		.params = wifi_config_policy,
 };
 
-static void rrm_set_basic_rate(const char *if_name, int32_t rate)
+static void rrm_set_mgmt_rate(const char *if_name, int32_t rate)
```
Comment (blogic) [inline feeds/wlan-ap/opensync/src/platform/openwrt/src/rrm/src/rrm_channel.c:39]: please remove this stub from the PR

Comment (yashvardhan-netex) [reply feeds/wlan-ap/opensync/src/platform/openwrt/src/rrm/src/rrm_channel.c:39]: This is something to be taken up soon. So did set it as a TODO. Nevertheless, Can remove it for now.

Comment (yashvardhan-netex) [reply feeds/wlan-ap/opensync/src/platform/openwrt/src/rrm/src/rrm_channel.c]: > please remove this stub from the PR

Done

**Review by RickSommerville — APPROVED** [review]
Looks good.  Thanks for clarification by chat.

**Comment by blogic** [comment]
merged into pending, Thanks !

**Pull request closed without merging**